CHANGELOG.md

```diff
@@ -592,6 +582,7 @@ which fields are to be projected is now determined automatically based on the me
 - Job and Batch classes for better simulation handling (eventually to fully replace webapi functions).
 - A large number of small improvements and bug fixes.
 
+[1.9.0]: https://github.com/flexcompute/tidy3d/compare/v1.8.4...v1.9.0
 [1.9.0rc2]: https://github.com/flexcompute/tidy3d/compare/v1.8.4...v1.9.0rc2
```

Comment:
are these rc1 and rc2 referenced anywhere above? if not maybe we should just remove them?